lesson_06/script.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
+const ul = document.getElementById("js-list");
+const fragment = document.createDocumentFragment();
+const data = [
+  { to: "bookmark.html", img: "1.png", alt: "画像1", text: "ブックマーク" },
+  { to: "message.html", img: "2.png", alt: "画像2", text: "メッセージ" }
+];
+
+const promise = new Promise((resolve) => {
+  setTimeout(() => {
+    resolve(data);
+  }, 3000);
+});
```

Comment:
```suggestion
const promise = new Promise((resolve) => {
 const data = [
  { to: "bookmark.html", img: "1.png", alt: "画像1", text: "ブックマーク" },
  { to: "message.html", img: "2.png", alt: "画像2", text: "メッセージ" }
];
  setTimeout(() => {
    resolve(data);
  }, 3000);
});
```
Nits
When I did this assignment, my teacher reviewed it like this.